app/pulp/app/apps.py

```diff
@@ -40,6 +40,9 @@ def __init__(self, app_name, app_module):
         self.named_viewsets = None
 
     def ready(self):
+        # register signals
+        import pulp.app.signals  # noqa
```

Comment:
[Django docs (see note)](https://docs.djangoproject.com/en/1.8/topics/signals/#connecting-receiver-functions) recommend to register signals in a  `ready` method of the app.
They also say that signals and their registration can live almost anywhere. I tried to follow Django recommendations though.